scripts/aggregate_result.py

```diff
@@ -116,9 +119,28 @@ def aggregate_results(model: str) -> Dict[str, float]:
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument("--model", required=True, type=str, help="Model name to aggregate")
+    parser.add_argument(
+        "--model", required=True, type=str, help="Model name to aggregate"
+    )
+    tmp_args, _ = parser.parse_known_args()
+    parser.add_argument(
+        "--result-dir",
+        type=str,
+        default=os.path.join("results", tmp_args.model),
+        help="Result directory",
+    )
+    parser.add_argument(
+        "--column-path-key-path",
+        type=str,
+        default="scripts/column-path-key.csv",
```

Comment:
```suggestion
        default=os.path.join("results", "column-path-key.csv"),
```
linuxでは差がないですが、一応linux以外でも使えるようにすました。